unisave-rest/src/main/java/org/uniprot/api/unisave/repository/domain/impl/ReleaseImpl.java

```diff
@@ -0,0 +1,50 @@
+package org.uniprot.api.unisave.repository.domain.impl;
+
+import lombok.Data;
+import org.uniprot.api.unisave.repository.domain.DatabaseEnum;
+import org.uniprot.api.unisave.repository.domain.Release;
+
+import javax.persistence.*;
+import java.util.Date;
+
+@Entity(name = "Release")
+@NamedQueries({
+    @NamedQuery(
+            name = "ReleaseImpl.findPastReleasesInOrder",
+            query =
+                    "select r from Release r where r.releaseDate <= :date order by r.releaseDate desc"),
+    @NamedQuery(
+            name = "ReleaseImpl.findReleaseByReleaseNumber",
+            query = "SELECT r from Release r where r.releaseNumber=:rel"),
+    @NamedQuery(
+            name = "ReleaseImpl.findAllRelease",
+            query = "SELECT r from Release r where r.id <> 9999999999 order by r.id desc")
+})
+@Data
+public class ReleaseImpl implements Release {
+
+    public enum Query {
+        findReleaseByReleaseNumber,
```

Comment:
sonar will complain about lower case enums, it is not a blocker, I think major only

Reply:
done